specification/security/resource-manager/Microsoft.Security/preview/2015-06-01-preview/security.json

```diff
@@ -1437,6 +1437,437 @@
                     }
                 }
             }
+        },
+        "/subscriptions/{subscriptionId}/providers/Microsoft.Security/northSouthHardenings": {
+            "get": {
+                "x-ms-examples": {
+                    "List all north-south hardening resources for a subscription": {
+                        "$ref": "./examples/NorthSouthHardenings/ListAllNorthSouthHardenings_example.json"
+                    }
+                },
+                "description": "Gets a list of north-south hardening resources for the subscription.",
+                "tags": [
+                    "NorthSouthHardenings"
+                ],
+                "operationId": "NorthSouthHardenings_ListAll",
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/ApiVersion"
+                    },
+                    {
+                        "$ref": "#/parameters/SubscriptionId"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "schema": {
+                            "$ref": "#/definitions/NorthSouthHardeningsList"
+                        }
+                    },
+                    "default": {
+                        "description": "Error response describing why the operation failed.",
+                        "schema": {
+                            "$ref": "#/definitions/CloudError"
+                        }
+                    }
+                },
+                "x-ms-pageable": {
+                    "nextLinkName": "nextLink"
+                }
+            }
+        },
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Security/northSouthHardenings": {
+            "get": {
+                "x-ms-examples": {
+                    "List north-south hardening resources for a resource group": {
+                        "$ref": "./examples/NorthSouthHardenings/ListNorthSouthHardenings_example.json"
+                    }
+                },
+                "description": "Gets a list of north-south hardening resources for the resource group.",
+                "tags": [
+                    "NorthSouthHardenings"
+                ],
+                "operationId": "NorthSouthHardenings_List",

```

Comment:
Naming convention for method that list resource in resource-group level is `ListByResourceGroup` so operation id should be `NorthSouthHardenings_ ListByResourceGroup`

Reply:
Done